### DISCUSSION TITLE
Supprimer le temps pour ecrire les commande SVP

### DISCUSSION THREAD

**Discussion by SmartFox1** [Crownicles/Crownicles#2466] [Nouvelles idées]
A chaque fois je dois taper la memes commande 4 fois pour que ca l'accept. Peut un port le temp que j'ecrit la commande. C'est tres enervent. Si vous pourait supprimer ca serait bien.

ici un exemple:
![1](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/165275188/15f56488-8240-4730-8455-977d6707d634)

**Comment by KirIcare** [comment]
Y'a pas de temps pour écrire les commandes, ça c'est juste une sorte de bug qui viens... De discord ou des shards je sais pas trop. À voir avec un mec plus compétent 😅

**Comment by KirIcare** [comment]
Enfin si y'a un temps mais vrm infime, assez infime pour ne pas être là cause du problème que tu rencontres
